src/components/modals/RoleModal.vue

```diff
@@ -11,15 +8,41 @@
           : "bluffing"
       }}
     </h3>
-    <ul class="tokens">
-      <li
-        v-for="role in availableRoles"
-        :class="[role.team]"
-        :key="role.id"
-        @click="setRole(role)"
-      >
-        <Token :role="role" />
+    <ul>
+      <li class="button-group" v-if="isBluffs()">
+        <span
+          class="button"
+          :class="{ active: tab === 'editionRoles' }"
+          @click="tab = 'editionRoles'"
+          >Edtition Roles</span
+        >
+        <span
+          class="button"
+          :class="{ active: tab === 'otherTravellers' }"
```

Comment:
I'd try to be consistent with the naming here, either call it `otherTravelers` or `extraTravelers` everywhere